### PR TITLE
Do a better job of sorting elements and behaviors in READMEs.

### PR DESCRIPTION
**Pull request by rictic**
If a repo is primarily about a behavior, we want to put that behavior
front and center in the README. This will effect app-storage and
iron-overlay-behavior.

Fixes https://github.com/PolymerElements/app-storage/issues/40

### CODE REVIEW

File: cleanup-passes/readme.ts [hunk 1]
```diff
@@ -77,60 +58,33 @@ thing! https://github.com/PolymerLabs/tedium/issues
...
+    // first.
+    readmeContents += docInfo.nameToContent.get(canonicalBehaviorName);
+    docInfo.nameToContent.delete(canonicalBehaviorName);
+  }
 
-${injectAutodetectedLanguage(behavior.desc)}
-`;
+  // Otherwise, it's the elements then the behaviors in sorted order.
+  const names = [
+      ...docInfo.tagNames].sort().concat([...docInfo.behaviorNames].sort())
```
Comment (justinfagnani) [inline cleanup-passes/readme.ts]: style nit: don't break after `[`, break after `]`

File: cleanup-passes/readme.ts [hunk 2]
```diff
@@ -145,6 +99,72 @@ ${injectAutodetectedLanguage(behavior.desc)}
   }
 }
 
+function extractDocumentationInfo(element: ElementRepo) {
```
Comment (justinfagnani) [inline cleanup-passes/readme.ts:102]: return type?

**Comment by justinfagnani** [comment]
LGTM

File: cleanup-passes/readme.ts (hunk 2 above)
Comment (rictic) [inline cleanup-passes/readme.ts:102]: It's a private function (called just once, and in this file) and the return type is complicated. My preference would be to let inference handle this one.

**Comment by rictic** [comment]
Thanks for the review!

**Pull request merged**